src/EpiContentUsage/Frontend/Styles/epi-content-usage.scss

```diff
@@ -1,8 +1,18 @@
+@use 'sass:map';
+@import "./utils";
+
 .epi-content-usage {
   position: relative;
 
   .epi-content-usage-grid {
+    .epi-content-usage-table-container {
+      overflow-x: auto;
+    }
+
     .epi-content-usage-table {
+      min-width: max-content;
```

Comment:
It makes this table too wide from the very beginning.

![image](https://github.com/fortedigital/EpiContentUsage/assets/31856712/4fd12e77-153d-46fa-8026-63312502145d)